gravitee-am-ui/src/app/domain/settings/resources/resource/resource.component.ts

```diff
@@ -89,10 +89,10 @@ export class ResourceComponent implements OnInit {
       });
   }
 
-  isHttpResource() {
+  isHttpFactorResource() {
     if (this.resourceSchema) {
       const resourceId = this.resourceSchema.id;
-      return resourceId ? resourceId.includes('HttpResourceConfiguration') : false;
+      return resourceId ? resourceId.includes('HttpFactorResourceConfiguration') : false;
```

Comment:
Why not test the resource `type` ? like the previous example : 

```
 return type ? type.includes('http-factor-am-resource') : false;
```

Reply:
changed